COMMUNICATION.md

```diff
@@ -1,58 +1,63 @@
+If you feel someone’s reaction is not clear enough (i.e. someone’s reaction confuses you, or even agitates you), look at these guidelines together to see what could be improved in the future. If you realise something critical is missing in the guidelines, please add to it.
+
+Try to keep in mind that written communication it’s really easy to misread something or to interpret something out of context. Try not to assume rudeness, ill intent or malice. Sometimes something was poorly phrased os misread. Sometimes we’re switching between contexts too often, and end up being shorter than we should. Try asking for clarification, or if the person could rephrase that, if the meaning is’t clear. 👍 
 
-If following hte guidelines don't clear things up consider a call to solve the issue if the matter is pressing, or wait for the next iOS meeting to continue discussing it. '
+If following the guidelines don't clear things up, consider a call to solve the issue if the matter is pressing, or wait for the next iOS meeting to continue discussing it.
 
 ### Good to keep in mind:
-These are the guidelines from GitHub for giving feedback on a PR, there's some valuable points in it for online communication in general:'
-
-> - Familiarize yourself with the context of the issue, and reasons why this Pull Request exists.
-> - If you disagree strongly, consider giving it a few minutes before responding; think before you react.
-> - Ask, don’t tell. (“What do you think about trying…?” rather than “Don’t do…”)
-> - Explain your reasons why code should be changed. (Not in line with the style guide? A personal preference?)
-> - Offer ways to simplify or improve code.
-> - Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced.
-> - Be humble. (“I’m not sure, let’s try…”)
-> - Avoid hyperbole. (“NEVER do…”)
-> - Aim to develop professional skills, group knowledge and product quality, through group critique.
-> - Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Can you use positive language as opposed to neutral?
-> - Use emoji to clarify tone. Compare “✨ ✨ Looks good 👍 ✨ ✨” to “Looks good.”
+These are paraphrased or downright copied over from Github’s pull request guideline:
+
+- Familiarise yourself with the context of the issue.
+- If you disagree strongly, consider giving it a few minutes before responding; think before you react. Try to empathise.
+- Ask, don’t tell. (“What do you think about trying…?” rather than “Don’t do…”)
+- Explain your reasons why code should be changed. (Not in line with the style guide? A personal preference?)
+- Offer ways to simplify or improve code.
+- Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced. Don’t shame, no one started out knowing.
+- Be humble. (“I’m not sure, let’s try…”)
+- NEVER use hyperboles. Unless needed. (“Never store sensitive data in plain text!”)
```

Comment:
I think the example you give here "Never store sensitive data in plain text!" is not necessarily a hyperbole. the internet gives me as definition for hyperbole: "A claim made with extreme exaggeration." So in the case of the gitHub example write "NEVER" in CAPS. I think that is something we should DEFINITELY avoid ;) With a hyperbole you tend to expres emotion, which I think we should avoid in professional discussions.

It might be a good idea to rewrite these guidelines from github to our own standards, so this example would be stating more clearly as to what a hyperbole is.